R/expectation.R

```diff
@@ -178,10 +172,15 @@ print.expectation <- function(x, ...) cat(format(x), "\n")
 #' @export
 format.expectation <- function(x, ...) {
```

Comment:
I wonder if we should a separate method for `success` (and I wonder if the class names should actually `expectation_success()`, `expectation_skipped()` etc)